pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -449,7 +450,9 @@ func (p *criStatsProvider) getFsInfo(fsID *runtimeapi.FilesystemIdentifier) *cad
 	fsInfo, err := p.cadvisor.GetDirFsInfo(mountpoint)
 	if err != nil {
 		msg := "Failed to get the info of the filesystem with mountpoint"
-		if err == cadvisorfs.ErrNoSuchDevice {
+		if err == cadvisorfs.ErrNoSuchDevice ||
```

Comment:
nit: should this also be the `errors.Is(` instead of equal?

Reply:
Yep this should work, changed as suggested :+1: